python/sglang/srt/openai_api/adapter.py

```diff
@@ -82,6 +82,17 @@ def __init__(self, filename: str, purpose: str):
 storage_dir = None
 
 
+def format_finish_reason(finish_reason) -> Optional[str]:
+    if finish_reason.startswith("None"):
+        return None
+    elif finish_reason.startswith("FINISH_MATCHED"):
+        return "stop"
+    elif finish_reason.startswith("FINISH_LENGTH"):
+        return "length"
+    else:
+        raise ValueError(f"Unknown finish reason: {finish_reason}")
```

Comment:
1. handle this https://github.com/sgl-project/sglang/blob/575f6fda0123428e41bbb11c20f4f3f36726f74f/python/sglang/srt/managers/schedule_batch.py#L83C7-L83C19?
1. return "unknown" to avoid crashing the server or the request?